src/Path/IO.hs

```diff
@@ -224,6 +226,12 @@ copyDirectoryRecursive srcDir destDir =
                   Nothing -> return ()
                   Just relSubDir -> copyDirectoryRecursive srcSubDir (destDir </> relSubDir))
 
+
+-- | Drop the root (either @\/@ on POSIX or @C:\\@, @D:\\@, etc. on
+-- Windows).
+dropRoot :: Path Abs t -> Path Rel t
```

Comment:
I know.  Chris suggested that I put it there for now. (slack chat) He said it was a staging area for possible Path additions.